### ISSUE TITLE
 Additional Binaries on 0.1.4 Release

### ISSUE THREAD

**Issue by UsaRandom** [dogecoinfoundation/libdogecoin#219]
In the upcoming 0.1.4 release of libdogecoin, please include the following additional binaries:

**Shared Libraries (DLLs):** Include shared libraries (DLLs) for Windows builds. This will allow developers to easily integrate libdogecoin into their projects without having to compile the library from source.
**Debug and Release Builds:** Provide separate debug and release builds. Debug builds have additional references and are more difficult to distribute. 
**Build with TPM Flag On and Off:** Include separate builds with the Trusted Platform Module (TPM) flag enabled and disabled. This is a useful security feature and should be part of the official release, even if it's only supported on windows.

Rationale:
These additional binaries and build options will make it easier for developers to use libdogecoin in their projects.

I personally have plans to polish up and publish Lib.Dogecoin (C# bindings) to NuGet (.NET package manager) and I would feel much better about doing that with official binaries. 

Please consider including these additions in the 0.1.4 release of libdogecoin.

**Comment by michilumin** [comment]
Yes, we can do that. We have .dll's building internally for use in RadioDoge, so we can add these to a binary package for the release process starting with 0.1.4.

**Comment by edtubbs** [comment]
@usarandom, #244 updates the ci to build Windows native and no TPM builds in release mode, with shared DLLs.

**Comment by edtubbs** [comment]
Resolved in `0.1.4`
